### PR TITLE
Fix incorrect var in IPA building

### PR DESCRIPTION
**Pull request by fmuyassarov**
Fix incorrect var in IPA building

### CODE REVIEW

**Review by Rozzii — APPROVED** [review]
Please merge this after: this https://github.com/Nordix/airship-dev-tools/pull/357 is already merged.

File: ci/scripts/image_scripts/start_centos_ipa_build.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ ssh \
   -o ServerAliveInterval=15 \
   -o ServerAliveCountMax=10 \
   -i "${AIRSHIP_CI_USER_KEY}" \
-  "${AIRSHIP_CI_USER}"@"${BUILD_EXECUTER_IP}" \
+  "${AIRSHIP_CI_USER}"@"${BUILDER_IP}" \
   PATH=/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/bin \
   "/tmp/${IPA_BUILDER_SCRIPT_NAME}" "${RT_USER}" "${RT_TOKEN}"
```
Comment (Rozzii) [inline ci/scripts/image_scripts/start_centos_ipa_build.sh:58]: Could you please move the RT_USER and RT_TOKEN before the /tmp/${IPA_BUILDER_SCRIPT_NAME}.
I guess this should work: RT_USER="${RT_USER}" RT_TOKEN="${RT_TOKEN}" /tmp/${IPA_BUILDER_SCRIPT_NAME}

**Review by Rozzii — CHANGES_REQUESTED** [review]

File: ci/scripts/image_scripts/start_centos_ipa_build.sh (hunk 1 above)
Comment (Rozzii) [reply ci/scripts/image_scripts/start_centos_ipa_build.sh:58]: I'm asking because I see that the build succeeds but the upload doesn't.

Comment (fmuyassarov) [reply ci/scripts/image_scripts/start_centos_ipa_build.sh:58]: Yep I also noticed that we are missing the in the buuld_ipa.sh. Please see the https://github.com/Nordix/airship-dev-tools/pull/356/commits/f8df0e5477a8957810661891153a83580b6198ff

Comment (Rozzii) [reply ci/scripts/image_scripts/start_centos_ipa_build.sh:58]: I wanted to pass then as environment variables not as parameters, but ofc we can use them as parametes although I think it is not advised to pass credentials as cli parameters.

Comment (fmuyassarov) [reply ci/scripts/image_scripts/start_centos_ipa_build.sh:58]: Ok sure, I can fix it. Give me a moment

**Review by Rozzii — APPROVED** [review]

**Pull request merged**